layout/main.scss

```diff
@@ -50,9 +50,9 @@
 	}
 
 	@include nUiCriticalEnd('core');
-
+	// needed to stop css minimize merging the selectors above and below
 	// undo the above hiding of the footer
-	.n-layout__row--footer {
+	.n-layout__row--footer.n-layout__row--footer {
```

Comment:
isn't it `.n-layout__row.n-layout__row--footer` ...? If the answer to that is now, then I'm confused 🙈